addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -736,14 +770,19 @@ def _prepare_bills_vals(self):
 
     def _prepare_move_vals(self):
         self.ensure_one()
-        return {
-            # force the name to the default value, to avoid an eventual 'default_name' in the context
-            # to set it to '' which cause no number to be given to the account.move when posted.
```

Comment:
If this comment is still true, we might want to keep it, especially because we are still giving a name.